src/main/java/com/partydefencetracker/DefenceTrackerPlugin.java

```diff
@@ -634,7 +643,15 @@ private void calculateDefence(SpecialWeapon weapon, int hit)
 				}
 				else
 				{
-					bossDef -= bossDef * .30;
+					if (client.getVarbitValue(Varbits.IN_RAID) == 1 && toaBossesWithDefenceLevels.containsKey(boss))
+					{
+						int actualBossDefence = toaBossesWithDefenceLevels.get(boss);
+						bossDef -= actualBossDefence * .30;
```

Comment:
While this gives the correct result, it's only because you can fully lower the bosses defence with a single spec. This line is computing 30% of the *maximum* defence instead of the *current* defence.